CHANGES.md

```diff
@@ -1,4 +1,39 @@
-# Changelog
+# Changelog 
+
+## 1.8.12 (core) / 0.24.12 (libraries)
+
+### New
+
+* The `AutomationCondition.eager()`, `AutomationCondition.missing()`, and `AutomationCondition.on_cron` conditions are now compatible with asset checks. [#25170](https://github.com/dagster-io/dagster/pull/25170)
```

Comment:
Historically, we've removed the links to the specific PRs

Reply:
Took these out. FWIW I think it would be useful to include these in the future.